kong.conf.default

```diff
@@ -213,6 +213,8 @@
                                  # preserved in the cache of each worker
                                  # process. When this number is exceeded, the
                                  # least recently used connections are closed.
+                                 # 0 will remove the entry conf nginx-kong.conf
+                                 # when using the default template
```

Comment:
Will you please update this description to something like:
```
A value of `0` will disable this behavior altogether,
forcing each upstream request to open a new connection.
```

`nginx-kong.conf` and "the default template" are implementation details that should not be mentioned here. Additionally, the proposed text is more explicit as to what the behavior is when the given value is `0`.